interfaces/mount/change.go

```diff
@@ -42,6 +44,29 @@ type Change struct {
 	Action Action
 }
 
+// Perform executes the desired mount or unmount change.
```

Comment:
It isn't about importing from all over, it is about putting the code where it belongs. The mount backend deals with mounts, yes, but it deals with them insofar as to create files for snap-confine to consume and not to perform mounts. If this doesn't resonate with you or other reviewers, I won't block on this point.

I do think that my comment regarding return syscall still holds. As written, a caller of Perform() may decide to not care about the error but since this is for snap-update-ns, it seems that you'd want to die here. Is there a particular reason you aren't doing this?